src/js/control-bar/volume-control/volume-bar.js

```diff
@@ -105,11 +105,19 @@ class VolumeBar extends Slider {
    * @listens Player#volumechange
    */
   updateARIAAttributes(event) {
-    // Current value of volume bar as a percentage
-    const volume = Math.round((this.player_.volume() * 100)).toString();
+    const ariaValue = this.player_.muted() ? 0 : this.volumeAsPercentage_();
 
-    this.el_.setAttribute('aria-valuenow', volume);
-    this.el_.setAttribute('aria-valuetext', volume + '%');
+    this.el_.setAttribute('aria-valuenow', ariaValue);
```

Comment:
Does it matter that you haven't converted `ariaValue` to a string by this point (where the previous code used the `.toString()` method on the numeric value)? According to [Using the aria-valuenow attribute - Accessibility | MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-valuenow_attribute), the value should be "_String representation of a number_". (The `aria-valuetext` will become a string thanks to the `+ '%'`)